src/backend/stubs/mail/create_account.stub

```diff
@@ -0,0 +1,312 @@
+                          <p
+                            style="
+                              border-top: solid 4px #002666;
+                              font-size: 1px;
+                              margin: 0px auto;
+                              width: 100%;
+                            "
+                          ></p>
+                          <!--[if mso | IE
+                            ]><table
+                              align="center"
+                              border="0"
+                              cellpadding="0"
+                              cellspacing="0"
+                              style="
+                                border-top: solid 4px #002666;
+                                font-size: 1px;
+                                margin: 0px auto;
+                                width: 550px;
+                              "
+                              role="presentation"
+                              width="550px"
+                            >
+                              <tr>
+                                <td style="height: 0; line-height: 0">&nbsp;</td>
+                              </tr>
+                            </table><!
+                          [endif]-->
+                        </td>
+                      </tr>
+                      <tr>
+                        <td
+                          align="left"
+                          style="
+                            font-size: 0px;
+                            padding: 10px 25px;
+                            word-break: break-word;
+                          "
+                        >
+                          <div
+                            style="
+                              font-family: helvetica;
+                              font-size: 20px;
+                              line-height: 1;
+                              text-align: left;
+                              color: #002666;
+                            "
+                          >
+                            Create your account
+                          </div>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td
+                          align="left"
+                          style="
+                            font-size: 0px;
+                            padding: 10px 25px;
+                            word-break: break-word;
+                          "
+                        >
+                          <div
+                            style="
+                              font-family: Ubuntu, Helvetica, Arial, sans-serif;
+                              font-size: 16px;
+                              line-height: 1;
+                              text-align: left;
+                              color: #000000;
+                            "
+                          >
+                            Hi \{\{ name \}\}, you have been invited to collaborate on the
```

Comment:
The spec says:
![Screenshot 2025-06-13 at 16 16 05](https://github.com/user-attachments/assets/6a6b69d9-a1eb-44c5-ba96-c83fcb803d59)

Reply:
Thank for this. Missed it.